app/models/rbac.rb

```diff
@@ -323,6 +323,11 @@ def self.get_user_info(user, userid, miq_group, miq_group_id)
   end
 
   def self.filtered(objects, options = {})
+    if objects.nil?
+      Vmdb::Deprecation.deprecation_warning("objects = nil",
```

Comment:
Well, our goal is to just have 1 field fro `klass`, `scope`, `conditions`, and possibly limits too. In the perfect world, we wouldn't even need a collection at all.

So for now, matching the results from `search` and not making unnecessary `exist?` calls to the database is a win.